scripts/parse_metanetx.py

```diff
@@ -64,13 +64,14 @@ def parse_coeff_and_metabolites(term):
     return stoichiometry
 
 
-def construct_universal_model(list_of_db_prefixes):
+def construct_universal_model(list_of_db_prefixes, reac_xref, reac_prop, chem_prop):
```

Comment:
perhaps it could afford a docstring as not immediately obvious what `reac_xref` should be..